torch/utils/data/datapipes/iter/combining.py

```diff
@@ -363,6 +397,11 @@ class ZipperIterDataPipe(IterDataPipe[Tuple[T_co]]):
 
     Args:
         *datapipes: Iterable DataPipes being aggregated
+
+    Example:
+        >>> from torchdata.datapipes.iter import IterableWrapper
+        >>> dp1, dp2, dp3 = IterableWrapper(range(5)), IterableWrapper(range(10, 15)), IterableWrapper(range(20, 25))
+        >>> list(dp1.zip(dp2, dp3))  # [(0, 10, 20), (1, 11, 21), (2, 12, 22), (3, 13, 23), (4, 14, 24)]
```

Comment:
Not related to this diff. For syntax sugar, we may want to provide another functional api for such operation `torchdata.datapipes.iter.zip(dp1, dp2, dp3)`